content/en/docs/concepts/configuration/secret.md

```diff
@@ -886,15 +887,29 @@ In this case, `0` means you have created an empty Secret.
 ### Service account token Secrets
 
 A `kubernetes.io/service-account-token` type of Secret is used to store a
-token that identifies a
+token credential that identifies a
 {{< glossary_tooltip text="service account" term_id="service-account" >}}.
+
+Since 1.22, this type of Secret is no longer used to mount credentials into Pods,
```

Comment:
So in 1.22 & 1.23 it made a secret with token but didn’t use it for the pods?

Reply:
Yes, to accommodate 2 version node skew, because pods scheduled on 1.21 nodes could still have required secret-based tokens if a cluster admin had disabled bound token projected volumes.

 1.22 is when the secrets injected to pods unconditionally used projected tokens.